presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -38,6 +38,22 @@ private MathFunctions() {}
 
     @Description("absolute value")
     @ScalarFunction("abs")
+    @SqlType(StandardTypes.TINYINT)
+    public static long absTinyInt(@SqlType(StandardTypes.TINYINT) long num)
```

Comment:
style note: the casing here isn't consistent with other places. It looks like we use `bigint` as a single word in other places, and you used `Smallint` in `SmallintType`, so I would stick with casing it as a single word